### ISSUE TITLE
single incorrect result for nearest edge

### ISSUE THREAD

**Issue by jGaboardi** [gboeing/osmnx#810]
**Problem description**
  - What did you do?
    - Attempted to find the nearest edge to a point. 
  - What did you expect to happen?
    - Return the nearest edge. 
  - And what actually happened instead?
    - Returns the incorrect edge only for a *specific point* (same edge every time).
    - This happens at multiple scales, but returns the correct nearest edge when the "troublemaker"(?) edge is not included. The following includes links to more detailed examples:
      - correct result with network instance [600 meters](https://gist.github.com/jGaboardi/f21109b0d48d1b9af75262592f5a11f4) from point
      - incorrect result with network instance [1200 meters](https://gist.github.com/jGaboardi/48d08a0d7b1db55412668ad15b9e125e) & [2400 meters](https://gist.github.com/jGaboardi/e4b0b189dff89850fe1a812e228e7ae6) from point
      - incorrect result with network instance of all Knox County, TN

**Environment information**
  - What operating system are you using?
    - macOS Big Sur v11.6.5 (20G527)
  - What Python version are you using?
    - 3.10.2
  - What OSMnx version are you using?
    - 1.1.2
  - Provide a complete list of your environment's packages and their versions (for example, run `conda list` or `pip list` then paste the output between the two "details" tags below)

<details>

```
# Name                    Version                   Build  Channel
actlike                   0.0.9                     dev_0    <develop>
affine                    2.3.0                      py_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
appnope                   0.1.2           py310h2ec42d9_2    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0          py310he24745e_1    conda-forge
asttokens                 2.0.5              pyhd8ed1ab_0    conda-forge
attrs                     21.4.0             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.10.0             pyha770c72_0    conda-forge
black                     22.1.0             pyhd8ed1ab_0    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
blosc                     1.21.0               he49afe7_0    conda-forge
boost-cpp                 1.74.0               hdbf7018_7    conda-forge
branca                    0.4.2              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                h0d85af4_6    conda-forge
brotli-bin                1.0.9                h0d85af4_6    conda-forge
brotlipy                  0.7.0           py310he24745e_1003    conda-forge
bzip2                     1.0.8                h0d85af4_4    conda-forge
c-ares                    1.18.1               h0d85af4_0    conda-forge
ca-certificates           2021.10.8            h033912b_0    conda-forge
cairo                     1.16.0            h9e0e54b_1010    conda-forge
cartopy                   0.20.2          py310he18ef84_4    conda-forge
cenpy                     1.0.0.post4                py_0    conda-forge
certifi                   2021.10.8       py310h2ec42d9_1    conda-forge
cffi                      1.15.0          py310hcc37b68_0    conda-forge
cfgv                      3.3.1              pyhd8ed1ab_0    conda-forge
cfitsio                   4.0.0                hb20e66c_0    conda-forge
charset-normalizer        2.0.12             pyhd8ed1ab_0    conda-forge
click                     8.0.4           py310h2ec42d9_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.7.2              pyhd8ed1ab_1    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
contextily                1.2.0              pyhd8ed1ab_0    conda-forge
coverage                  6.3.2           py310h1961e1f_1    conda-forge
cryptography              36.0.1          py310ha82f1d4_0    conda-forge
curl                      7.82.0               h9f20792_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
cython                    0.29.26                  pypi_0    pypi
dataclasses               0.8                pyhc8e2a94_3    conda-forge
dbus                      1.13.6               h811a1a6_3    conda-forge
debugpy                   1.5.1           py310hba3363e_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
distlib                   0.3.4              pyhd8ed1ab_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
execnet                   1.9.0              pyhd8ed1ab_0    conda-forge
executing                 0.8.3              pyhd8ed1ab_0    conda-forge
expat                     2.4.7                h96cf925_0    conda-forge
filelock                  3.6.0              pyhd8ed1ab_0    conda-forge
fiona                     1.8.21          py310h56b70e1_0    conda-forge
flit-core                 3.7.1              pyhd8ed1ab_0    conda-forge
folium                    0.12.1.post1       pyhd8ed1ab_1    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.96              h676cef8_2    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.30.0          py310h1961e1f_0    conda-forge
freetype                  2.10.4               h4cff582_1    conda-forge
freexl                    1.0.6                h0d85af4_0    conda-forge
fuzzywuzzy                0.18.0             pyhd8ed1ab_0    conda-forge
gdal                      3.4.1                    pypi_0    pypi
geographiclib             1.52               pyhd8ed1ab_0    conda-forge
geopandas                 0.10.2             pyhd8ed1ab_1    conda-forge
geopandas-base            0.10.2             pyha770c72_1    conda-forge
geoplot                   0.5.0              pyhd8ed1ab_0    conda-forge
geopy                     2.2.0              pyhd8ed1ab_0    conda-forge
geos                      3.10.2               he49afe7_0    conda-forge
geotiff                   1.7.0                had63758_7    conda-forge
gettext                   0.19.8.1          hd1a6beb_1008    conda-forge
giflib                    5.2.1                hbcb3906_2    conda-forge
hdf4                      4.2.15               hefd3b78_3    conda-forge
hdf5                      1.12.1          nompi_ha60fbc9_104    conda-forge
icu                       69.1                 he49afe7_0    conda-forge
identify                  2.4.11             pyhd8ed1ab_0    conda-forge
idna                      3.3                pyhd8ed1ab_0    conda-forge
importlib-metadata        4.11.3          py310h2ec42d9_0    conda-forge
importlib_resources       5.4.0              pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
ipykernel                 6.9.1           py310ha188af9_0    conda-forge
ipython                   8.1.1           py310h2ec42d9_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.6.5              pyhd8ed1ab_0    conda-forge
jbig                      2.1               h0d85af4_2003    conda-forge
jedi                      0.18.1          py310h2ec42d9_0    conda-forge
jinja2                    3.0.3              pyhd8ed1ab_0    conda-forge
joblib                    1.1.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   h0d85af4_0    conda-forge
json-c                    0.15                 hcb556a6_0    conda-forge
jsonschema                4.4.0              pyhd8ed1ab_0    conda-forge
jupyter                   1.0.0           py310h2ec42d9_7    conda-forge
jupyter_client            7.1.2              pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.3              pyhd8ed1ab_0    conda-forge
jupyter_core              4.9.2           py310h2ec42d9_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_widgets        1.0.2              pyhd8ed1ab_0    conda-forge
kealib                    1.4.14               ha22a8b1_3    conda-forge
kiwisolver                1.3.2           py310h2fea185_1    conda-forge
krb5                      1.19.2               hb49756b_4    conda-forge
lcms2                     2.12                 h577c468_0    conda-forge
lerc                      3.0                  he49afe7_0    conda-forge
libblas                   3.9.0           13_osx64_openblas    conda-forge
libbrotlicommon           1.0.9                h0d85af4_6    conda-forge
libbrotlidec              1.0.9                h0d85af4_6    conda-forge
libbrotlienc              1.0.9                h0d85af4_6    conda-forge
libcblas                  3.9.0           13_osx64_openblas    conda-forge
libclang                  13.0.1          default_he082bbe_0    conda-forge
libcurl                   7.82.0               h9f20792_0    conda-forge
libcxx                    13.0.1               hc203e6f_0    conda-forge
libdap4                   3.20.6               h3e144a0_2    conda-forge
libdeflate                1.10                 h0d85af4_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.4.2                h0d85af4_5    conda-forge
libgdal                   3.4.1                h52a0caa_6    conda-forge
libgfortran               5.0.0           9_3_0_h6c81a4c_23    conda-forge
libgfortran5              9.3.0               h6c81a4c_23    conda-forge
libglib                   2.70.2               hf1fb8c0_4    conda-forge
libiconv                  1.16                 haf1e3a3_0    conda-forge
libkml                    1.3.0             h8fd9edb_1014    conda-forge
liblapack                 3.9.0           13_osx64_openblas    conda-forge
libllvm13                 13.0.1               h64f94b2_2    conda-forge
libnetcdf                 4.8.1           nompi_h6609ca0_101    conda-forge
libnghttp2                1.47.0               h942079c_0    conda-forge
libopenblas               0.3.18          openmp_h3351f45_0    conda-forge
libpng                    1.6.37               h7cec526_2    conda-forge
libpq                     14.2                 hea3049e_0    conda-forge
libpysal                  4.6.2              pyhd8ed1ab_0    conda-forge
librttopo                 1.1.0                hec60dd8_9    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libspatialindex           1.9.3                he49afe7_4    conda-forge
libspatialite             5.0.1               hadde3e2_15    conda-forge
libssh2                   1.10.0               h52ee1ee_2    conda-forge
libtiff                   4.3.0                h17f2ce3_3    conda-forge
libwebp                   1.2.2                h28dabe5_0    conda-forge
libwebp-base              1.2.2                h0d85af4_1    conda-forge
libxcb                    1.13              h0d85af4_1004    conda-forge
libxml2                   2.9.12               h7e28ab6_1    conda-forge
libzip                    1.8.0                h8b0c345_1    conda-forge
libzlib                   1.2.11            h9173be1_1013    conda-forge
llvm-openmp               13.0.1               hcb1a161_1    conda-forge
lz4-c                     1.9.3                he49afe7_1    conda-forge
mapclassify               2.4.3              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.0           py310h1961e1f_1    conda-forge
matplotlib                3.5.1           py310h2ec42d9_0    conda-forge
matplotlib-base           3.5.1           py310hfc59cd6_0    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
matplotlib-scalebar       0.8.0              pyhd8ed1ab_0    conda-forge
mercantile                1.2.1              pyhd8ed1ab_0    conda-forge
mip                       1.13.0                   pypi_0    pypi
mistune                   0.8.4           py310he24745e_1005    conda-forge
munch                     2.5.0                      py_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
mypy_extensions           0.4.3           py310h2ec42d9_4    conda-forge
mysql-common              8.0.28               h694c41f_0    conda-forge
mysql-libs                8.0.28               h115446f_0    conda-forge
nb_conda_kernels          2.3.1           py310h2ec42d9_1    conda-forge
nbclient                  0.5.13             pyhd8ed1ab_0    conda-forge
nbconvert                 6.4.4           py310h2ec42d9_0    conda-forge
nbformat                  5.2.0              pyhd8ed1ab_0    conda-forge
ncurses                   6.3                  he49afe7_0    conda-forge
nest-asyncio              1.5.4              pyhd8ed1ab_0    conda-forge
networkx                  2.6.3              pyhd8ed1ab_1    conda-forge
nodeenv                   1.6.0              pyhd8ed1ab_0    conda-forge
notebook                  6.4.8              pyha770c72_0    conda-forge
nspr                      4.32                 hcd9eead_1    conda-forge
nss                       3.74                 h31e2bf1_0    conda-forge
numexpr                   2.8.0           py310hdd25497_1    conda-forge
numpy                     1.22.2          py310hfbbbacf_0    conda-forge
openjpeg                  2.4.0                h6e7aa92_1    conda-forge
openssl                   1.1.1l               h0d85af4_0    conda-forge
osmnx                     1.1.2              pyhd8ed1ab_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandana                   0.6.1           py310h700695d_1    conda-forge
pandas                    1.4.1           py310hdd25497_0    conda-forge
pandoc                    2.17.1.1             h694c41f_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pathspec                  0.9.0              pyhd8ed1ab_0    conda-forge
pcre                      8.45                 he49afe7_0    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.0.1           py310h382dc3b_2    conda-forge
pip                       22.0.4             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               hbcb3906_0    conda-forge
platformdirs              2.5.1              pyhd8ed1ab_0    conda-forge
pluggy                    1.0.0           py310h2ec42d9_2    conda-forge
poppler                   22.01.0              h39497b0_1    conda-forge
poppler-data              0.4.11               hd8ed1ab_0    conda-forge
postgresql                14.2                 he8fe76e_0    conda-forge
pre-commit                2.17.0          py310h2ec42d9_0    conda-forge
proj                      9.0.0                h2364a93_1    conda-forge
prometheus_client         0.13.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.27             pyha770c72_0    conda-forge
prompt_toolkit            3.0.27               hd8ed1ab_0    conda-forge
pthread-stubs             0.4               hc929b4f_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pulp                      2.6.0                    pypi_0    pypi
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py                        1.11.0             pyh6c4a22f_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygeos                    0.12.0          py310hbb63119_1    conda-forge
pygments                  2.11.2             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.7              pyhd8ed1ab_0    conda-forge
pyproj                    3.3.0           py310h8089d31_2    conda-forge
pyqt                      5.12.3          py310h2ec42d9_8    conda-forge
pyqt-impl                 5.12.3          py310hea9ccf3_8    conda-forge
pyqt5-sip                 4.19.18         py310hd046c34_8    conda-forge
pyqtchart                 5.12            py310hea9ccf3_8    conda-forge
pyqtwebengine             5.12.1          py310hea9ccf3_8    conda-forge
pyrsistent                0.18.1          py310he24745e_0    conda-forge
pyshp                     2.2.0              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1           py310h2ec42d9_4    conda-forge
pytables                  3.7.0           py310hdbf068a_0    conda-forge
pytest                    7.0.1           py310h2ec42d9_0    conda-forge
pytest-cov                3.0.0              pyhd8ed1ab_0    conda-forge
pytest-forked             1.4.0              pyhd8ed1ab_0    conda-forge
pytest-xdist              2.5.0              pyhd8ed1ab_0    conda-forge
python                    3.10.2          h1dd9edd_4_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-levenshtein        0.12.2          py310he24745e_1    conda-forge
python_abi                3.10                    2_cp310    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pyyaml                    6.0             py310he24745e_3    conda-forge
pyzmq                     22.3.0          py310hcce64c8_1    conda-forge
qt                        5.12.9               h2a607e2_5    conda-forge
qtconsole                 5.2.2              pyhd8ed1ab_1    conda-forge
qtconsole-base            5.2.2              pyhd8ed1ab_1    conda-forge
qtpy                      2.0.1              pyhd8ed1ab_0    conda-forge
rasterio                  1.2.10          py310h7a5eec6_5    conda-forge
readline                  8.1                  h05e3726_0    conda-forge
requests                  2.27.1             pyhd8ed1ab_0    conda-forge
rtree                     0.9.7           py310had9ce37_3    conda-forge
scikit-learn              1.0.2           py310h598de7d_0    conda-forge
scipy                     1.8.0           py310h47774c9_1    conda-forge
seaborn-base              0.11.2             pyhd8ed1ab_0    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                60.9.3          py310h2ec42d9_0    conda-forge
shapely                   1.8.0           py310hbb63119_5    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snuggs                    1.4.7                      py_0    conda-forge
soupsieve                 2.3.1              pyhd8ed1ab_0    conda-forge
sqlite                    3.37.0               h23a322b_0    conda-forge
stack_data                0.2.0              pyhd8ed1ab_0    conda-forge
terminado                 0.13.3          py310h2ec42d9_0    conda-forge
testpath                  0.6.0              pyhd8ed1ab_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tiledb                    2.7.1                h190f7ed_0    conda-forge
tk                        8.6.12               h5dbffcc_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
tornado                   6.1             py310he24745e_2    conda-forge
tqdm                      4.63.0             pyhd8ed1ab_0    conda-forge
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
typed-ast                 1.5.2           py310he24745e_0    conda-forge
typing_extensions         4.1.1              pyha770c72_0    conda-forge
tzcode                    2021e                h0d85af4_0    conda-forge
tzdata                    2021e                he74cb21_0    conda-forge
ukkonen                   1.0.1           py310h2fea185_1    conda-forge
unicodedata2              14.0.0          py310he24745e_0    conda-forge
urllib3                   1.26.8             pyhd8ed1ab_1    conda-forge
versioneer                0.21               pyhd8ed1ab_0    conda-forge
virtualenv                20.13.3         py310h2ec42d9_0    conda-forge
watermark                 2.3.0              pyhd8ed1ab_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
widgetsnbextension        3.5.2           py310h2ec42d9_1    conda-forge
xerces-c                  3.2.3                h6564042_4    conda-forge
xorg-libxau               1.0.9                h35c211d_0    conda-forge
xorg-libxdmcp             1.1.3                h35c211d_0    conda-forge
xyzservices               2022.3.0           pyhd8ed1ab_0    conda-forge
xz                        5.2.5                haf1e3a3_1    conda-forge
yaml                      0.2.5                h0d85af4_2    conda-forge
zeromq                    4.3.4                he49afe7_1    conda-forge
zipp                      3.7.0              pyhd8ed1ab_1    conda-forge
zlib                      1.2.11            h9173be1_1013    conda-forge
zstd                      1.5.2                h582d3a0_0    conda-forge 
```

</details>

**Provide a complete minimal reproducible example**

Your example code snippet here must be *minimal* so it doesn't contain extraneous code unrelated to your specific problem and it must be *complete* so we can independently run it from top to bottom by copying/pasting it into a Python interpreter. That means all imports and all variables must be defined. If you're unsure how to create a good reproducible example, read [this guide](https://stackoverflow.com/help/minimal-reproducible-example). Do not post a screenshot of your code or error message: provide it as text.

```python
Python 3.10.2 | packaged by conda-forge | (main, Mar  8 2022, 15:58:48) [Clang 11.1.0 ]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.1.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: %paste
import geopandas
import osmnx
import shapely

# network event point in EPSG:3857
xy3875 = [-9344074.987531548365951, 4300687.976071485318244]
pnt3875 = shapely.geometry.Point(xy3875)
point_df = geopandas.GeoDataFrame(geometry=[pnt3875], crs="EPSG:3857")

# graph from EPSG:4326 transformed point
from_point = point_df.to_crs("EPSG:4326").loc[0].geometry.coords[0][::-1]
graph = osmnx.graph_from_point(
    from_point, dist=1200, network_type="drive", simplify=True
)
graph = osmnx.project_graph(graph, to_crs="EPSG:3857")
graph = osmnx.consolidate_intersections(graph, dead_ends=True)
graph = osmnx.utils_graph.get_largest_component(graph, strongly=True)
n, e = osmnx.graph_to_gdfs(graph)

# locate nearest edge - (124, 122, 0)
point_xys = point_df.geometry.map(lambda pt: (pt.x, pt.y))
observed_nearest = osmnx.nearest_edges(graph, *point_xys[0])
print(observed_nearest)

# known nearest edge - (16, 18, 0)
hiawassee = (e["u_original"]==202855045) & (e["v_original"]==202859564)
expected_nearest = e.loc[hiawassee].squeeze().name
print(expected_nearest)

observed_nearest == expected_nearest

## -- End pasted text --
(124, 122, 0)
(16, 18, 0)
Out[1]: False

```

cc @jvtcl

**Comment by gboeing** [comment]
Thanks for flagging this. Looks like a bug.

Note that if you use the `interpolate` parameter, you get the correct result. In your example, you're not using the `interpolate` parameter, so the function uses an r-tree to identify the nearest edge. It appears that the bounding rectangles of OSM ways [268678767](https://www.openstreetmap.org/way/268678767) and [19527434](https://www.openstreetmap.org/way/19527434) both intersect the point you're querying. The r-tree returns the single nearest geometry, which in this case arbitrarily chooses between these two edges. However, although their bounding rectangles both intersect your query point, one of the edges's line geometry is ~500m from your point and the other's is ~4m from your point. So the arbitrary choice fails us as it's picking the one that's ~500m away.

I think the solution would be to increase the `k` in how many `k`-nearest objects we're getting before we perform the distance minimization. That would change [this line](https://github.com/gboeing/osmnx/blob/main/osmnx/distance.py#L288) to something like:
```python
dists = geoms.iloc[list(rtree.nearest(xy, num_results=10))].distance(Point(xy))
```

**Comment by jGaboardi** [comment]
@gboeing excellent troubleshooting. I will try to get a PR in for this fix in the next couple days.

**Comment by jGaboardi** [comment]
Just tried your proposed fix and worked :tada:. Would it make sense to add a keyword argument for this defaulted to 10 (or something)?

```python
def nearest_edges(G, X, Y, interpolate=None, return_dist=False, k_tiebreakers=10): 
    ...
    dists = geoms.iloc[list(rtree.nearest(xy, k_tiebreakers=10))].distance(Point(xy))
    ...
```

**Comment by gboeing** [comment]
> Would it make sense to add a keyword argument for this

I had wondered about this myself. My general inclination is to say no, 1) to keep the API simple and 2) because I'm guessing that parameterizing it to something like 10 should resolve this issue for foreseeable cases. I'm also looking to drop the Rtree dependency once the ongoing GeoPandas / Shapely / pyGEOS work gets settled.

**Comment by gboeing** [comment]
Resolved by #813
